packages/nextjs/services/mongodb/seed.sample.json

```diff
@@ -21840,34 +21840,39 @@
     },
     {
       "label": "Gas Fees",
+      "longLabel": "Gas Fees",
       "name": "gas_fees",
       "description": "Sum of a project's total contribution to gas fees across the Superchain. Gas fees are the primary recurring revenue source for the Superchain and a key indicator of aggregate blockspace demand. A project’s gas fee contribution is influenced by its total volume of contract interactions, the computational complexity of those interactions, and the state of the underlying gas market at the time of those transactions. In the long run, gas fees are what will power Retro Funding and enable it to continue in perpetuity. All members of the Superchain have committed at least 15% of their gross profit from gas fees to Retro Funding. Supporting projects that generate revenue in the form of gas fees helps power the economic engine of the Superchain.",
       "activated": true,
       "order": 2
     },
     {
       "label": "Total Transactions",
+      "longLabel":"Total Transactions",
       "name": "transaction_count",
       "description": "Count of a project’s transactions over the RF4 scope period October 2023 - June 2024. Optimism is a Layer 2 roll-up designed to improve the transaction throughput and reduce the fees on Ethereum. Layer 2s are crucial for scaling Ethereum because they help address the network's congestion issues without compromising its security or decentralization. Transaction counts are an important indicator for assessing the adoption and usage of all the new blockspace made available by the Superchain. Projects that have a sustained, high transaction count provide a clear signal of network growth and blockspace demand.",
       "activated": true,
       "order": 3
     },
     {
-      "label": "Interactions from Trusted Optimism Users",
+      "label": "Optimism Users Interactions",
+      "longLabel":"Interactions from Trusted Optimism Users",
       "name": "trusted_transaction_count",
       "description": "Count of a project’s transactions performed by trusted users, over the RF4 scope period October 2023 - June 2024. Bots, airdrop farming, and sybil attacks are longstanding problems in crypto. There are several teams in the Optimism ecosystem building reputation models for labeling “trusted users” in a privacy-preserving way. This metric aggregates reputation data from multiple platforms ([Farcaster](https://docs.farcaster.xyz/learn/architecture/hubs), [Passport](https://www.passport.xyz/), [EigenTrust by Karma3Labs](https://docs.karma3labs.com/eigentrust)) and only considers transactions that come from trusted users, a small subset of all active addresses on the Superchain. By tracking interactions specifically from trusted users, we gain a picture of blockspace demand that is less influenced by the effects of bots / farmers / sybils.",
       "activated": true,
       "order": 4
     },
     {
-      "label": "Trusted Optimism Users' Share of Total Interactions",
+      "label": "Total Interactions",
```

Comment:
Maybe this should be "Trusted Share of Interactions" or something, because it represents a percentage, rather than a total

Reply:
done